server/views/home.njk

```diff
@@ -0,0 +1,61 @@
+{% extends 'layout.njk' %}
+
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Home",
+        href: "#"
+      }]
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">FFC Calculation Engine</h1>      
+
+      <p class="govuk-body">
+        This is a demo application for the FFC Calculation Engine.
+      </p>
+
+      {{ govukWarningText({
+        text: "This is not a real service and is for demonstration and investigation purposes only.",
+        iconFallbackText: "Warning"
+      }) }}
```

Comment:
I think this should be different from the User front end. Just some mention of "Admin interface" see we can tell that it's a different service from the other.